crates/context_attribute/src/lib.rs

```diff
@@ -91,13 +94,13 @@ pub fn context(_attributes: TokenStream, input: TokenStream) -> TokenStream {
                             "expected exactly two or three generic parameters"
                         ),
                     },
-                    "Parameter" | "PersistentState" => match &mut first_segment.arguments {
+                    "CyclerState" | "Parameter" => match &mut first_segment.arguments {
                         PathArguments::AngleBracketed(arguments) if arguments.args.len() == 2 => {
                             pop_string_argument(arguments);
                             let data_type = get_data_type(arguments);
                             into_reference_with_lifetime(
                                 data_type,
-                                (first_segment.ident == "PersistentState").then(Mut::default),
+                                (first_segment.ident != "Parameter").then(Mut::default),
```

Comment:
Why did you change this comparison?
Previously it said that PersistentState requires a mutable reference. Now it says that parameters don't.
But from just reading the code, no statement is made about whether the Persistent-/CyclerState require it, that information is implicit.